hpy/tools/autogen/public_api.h

```diff
@@ -963,6 +963,15 @@ HPy HPyTuple_FromArray(HPyContext *ctx, const HPy items[], HPy_ssize_t n);
  *
  * :param ctx:
  *     The execution context.
+ * 
+ * :param start:
+ *     A pointer to a variable where to write the unpacked slice start. Must not
+ *     be ``NULL``.
```

Comment:
I guess this is just a copy-and-paste-error but those parameters are not *pointers*.
I would write something like: "A handle to an object to be used as slice start" or something (your English is better than mine :slightly_smiling_face:). Also, we allow `HPy_NULL` !
(Same for the other params.)